app/models/config_path.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ConfigPath
+  EMPTY_PATH = '/'
+
+  attr_reader :path
+
+  def initialize(value)
+    self.path = value
+  end
+
+  def to_regex
+    return '/.*' if empty?
+
+    "#{path}/.*|#{path}/?"
+  end
+
+  def empty?
+    path == EMPTY_PATH
+  end
+
+  private
+
+  def path=(value)
+    @path = value.dup
+    @path = remove_duplicated_slashes(@path)
+    @path.insert(0, EMPTY_PATH) unless @path.start_with?(EMPTY_PATH)
+  end
```

Comment:
Can be simplified by:

```suggestion
  def path=(value)
    super(EMPTY_PATH + StringUtils::strip_slashes(value))
  end
```